src/utils/getCategoriesOption.js

```diff
@@ -1,8 +1,10 @@
 const getCategoriesOptions = (categories) => {
-  if (categories) {
+  if (Array.isArray(categories)) {
     return categories.map((item) => (item === 'null' ? null : item))
```

Comment:
Maybe it would be more logical to put this code inside of "if (categories)" block. But this version should work anyway.

Reply:
the problem is that categories can sometimes be string, and mapping string is impossible, so i need to check if categories is array to map it